src/main/java/sirius/biz/codelists/CodeLists.java

```diff
@@ -191,6 +191,33 @@ public String getRequiredValue(@Nonnull String codeList, @Nonnull String code) {
         return cle.getCodeListEntryData().getValue();
     }
 
+    /**
+     * Checks if the given code exists inside the given code list or throws an exception if no matching entry exists
+     *
+     * @param codeList the code list to search in
+     * @param code     the code to lookup
+     * @throws sirius.kernel.health.HandledException if no entry exists for the given code or code list
+     */
+    public void verifyValue(@Nonnull String codeList, @Nonnull String code) {
+        Optional<L> cl = findCodelist(codeList);
```

Comment:
Pls use a more descriptive variable name. Parameter Name could be `codeListName` so this could be called `codeList` maybe